community/go/set-external-linker.patch

```diff
@@ -0,0 +1,11 @@
+--- go/src/cmd/link/internal/amd64/obj.go.orig	2017-03-06 23:55:57.000000000 +0000
++++ go/src/cmd/link/internal/amd64/obj.go	2017-03-06 23:56:16.000000000 +0000
+@@ -67,7 +67,7 @@ func Init() {
+ 	ld.Thearch.Append64 = ld.Append64l
+ 	ld.Thearch.TLSIEtoLE = tlsIEtoLE
+ 
+-	ld.Thearch.Linuxdynld = "/lib64/ld-linux-x86-64.so.2"
++	ld.Thearch.Linuxdynld = "/lib/ld-musl-x86_64.so.1"
```

Comment:
If you need to patch the ld name here, you likely need to patch it for all supported architectures.